core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalConfiguration.java

```diff
@@ -53,6 +58,30 @@ public static RaftJournalConfiguration defaults(ServiceType serviceType) {
         .setPath(new File(JournalUtils.getJournalLocation().getPath()));
   }
 
+  /**
+   * @param clusterAddresses addresses of all nodes in the Raft cluster
+   * @return true if clusterAddress have an object of same ip with local Address, otherwise false
+   */
+  private boolean checkContainIp(List<InetSocketAddress> clusterAddresses) {
+    boolean exist = false;
+    String clusterItemIp;
+    String localAddressIp =  NetworkAddressUtils.getLocalIpAddress((int) ServerConfiguration
+        .global().getMs(PropertyKey.NETWORK_HOST_RESOLUTION_TIMEOUT_MS));
+    try {
+      for (InetSocketAddress addr : clusterAddresses) {
+        clusterItemIp = InetAddress.getByName(addr.getHostName()).getHostAddress();
+        if (clusterItemIp.equals(localAddressIp)) {
+          exist = true;
+          break;
+        }
+      }
+    } catch (UnknownHostException e) {
+      LOG.error("Get raft cluster node ip by hostname failed", e);
```

Comment:
is it possible that cluster addresses contain host A, host B, host C.
Host B is the local host and can be resolved, but the method error out when resolving host A?
will it cause issues in this case

Reply:
@LuQQiu Thanks for the suggestion. I think this may happen and cause error, I will check again.